whiteBoarding/bubbleSort/whiteboardingcode

```diff
@@ -0,0 +1,29 @@
+// Whiteboarding Question, Monday, Oct. 30, 2018
+// Given an array, sort the array using bubbleSort and then return the array.
+
+function myBubble[arr] => {
+    var inspect = ( );
+    var arraylenth = arr.lenth;
+    var arrIndex = arr;
+    if (inspect = false;
+    for (var i = 0; i < arraylenth.length +1; i++)
+       {if arrIndex[i] < arrIndex[i+1])
+           {var tempArra = arrIndex[i];
+            arrIndex[i] = arrIndex[i+1];
+            arrIndex[i+1] = tempArra;
+            inspect = true;}}
+    arraylenth ++;
+}
```

Comment:
To many curly braces here, It would be good to run this code in a compile and see where your syntax errors are...